layers/+lang/haskell/packages.el

```diff
@@ -54,6 +55,11 @@
   (use-package ghc
     :defer t))
 
+(defun haskell/init-dante ()
+  (use-package dante
+    :defer t
+    :init (add-hook 'haskell-mode-hook 'dante-mode)))
```

Comment:
Final question: Shouldn't enabling `dante-mode` in a specific haskell buffer be the responsibility of the `setup-dante` function?